patches/tModLoader/Terraria/GameContent/TownNPCProfiles.cs.patch

```diff
@@ -0,0 +1,60 @@
+--- src/TerrariaNetCore/Terraria/GameContent/TownNPCProfiles.cs
++++ src/tModLoader/Terraria/GameContent/TownNPCProfiles.cs
+@@ -1,4 +_,6 @@
+ using System.Collections.Generic;
++using Terraria.ID;
++using Terraria.ModLoader;
+ 
+ namespace Terraria.GameContent
+ {
+@@ -29,32 +_,48 @@
+ 			43,
+ 			44
+ 		};
++		// does Old Man count? should he? need to reference vanilla or ask someone who's done so already     -thomas
+ 		private Dictionary<int, ITownNPCProfile> _townNPCProfiles = new Dictionary<int, ITownNPCProfile> {
+ 			{ 369, new Profiles.LegacyNPCProfile("Images/TownNPCs/Angler", 22) },
++			{ NPCID.ArmsDealer, new Profiles.AlternateLegacyNPCProfile("Images/NPC_19", NPCHeadID.ArmsDealer) },
+ 			{ 633, new Profiles.TransformableNPCProfile("Images/TownNPCs/BestiaryGirl", 26) },
+ 			{ 54, new Profiles.LegacyNPCProfile("Images/TownNPCs/Clothier", 7) },
+ 			{ 209, new Profiles.LegacyNPCProfile("Images/TownNPCs/Cyborg", 16) },
++			{ NPCID.DD2Bartender, new Profiles.AlternateLegacyNPCProfile("Images/NPC_550", NPCHeadID.DD2Bartender) },
+ 			{ 38, new Profiles.LegacyNPCProfile("Images/TownNPCs/Demolitionist", 4) },
++			{ NPCID.Dryad, new Profiles.AlternateLegacyNPCProfile("Images/NPC_20", NPCHeadID.Dryad) },
+ 			{ 207, new Profiles.LegacyNPCProfile("Images/TownNPCs/DyeTrader", 14) },
++			{ NPCID.GoblinTinkerer, new Profiles.AlternateLegacyNPCProfile("Images/NPC_107", NPCHeadID.GoblinTinkerer) },
+ 			{ 588, new Profiles.LegacyNPCProfile("Images/TownNPCs/Golfer", 25) },
++			{ NPCID.Guide, new Profiles.AlternateLegacyNPCProfile("Images/NPC_22", NPCHeadID.Guide) },
+ 			{ 124, new Profiles.LegacyNPCProfile("Images/TownNPCs/Mechanic", 8) },
+ 			{ 17, new Profiles.LegacyNPCProfile("Images/TownNPCs/Merchant", 2) },
+ 			{ 18, new Profiles.LegacyNPCProfile("Images/TownNPCs/Nurse", 3) },
+ 			{ 227, new Profiles.LegacyNPCProfile("Images/TownNPCs/Painter", 17) },
++			{ NPCID.PartyGirl, new Profiles.AlternateLegacyNPCProfile("Images/NPC_208", NPCHeadID.PartyGirl) },
+ 			{ 229, new Profiles.LegacyNPCProfile("Images/TownNPCs/Pirate", 19) },
++			{ 663, new Profiles.LegacyNPCProfile("Images/TownNPCs/Princess", 45) },
```

Comment:
Is there a reason why this line was moved? Accidental change?

Reply:
I had wanted to organize them alphabetically but now that I think about it that falls apart with the town pets so I might just move Princess back to her original spot